### PR TITLE
Polish actor and statement types

### PR DESCRIPTION
**Pull request by Selindek**

### CODE REVIEW

File: projects/ngx-xapi/model/src/lib/actor.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ type ActorType = {
 
 type IdentifiedActor =
   | {
-      mbox: string;
+      mbox: `mailto:${string}`;
```
Comment (thomasturrell) [inline projects/ngx-xapi/model/src/lib/actor.ts:10]: wow!

File: projects/ngx-xapi/model/src/lib/statement.ts [hunk 2]
```diff
@@ -24,7 +25,7 @@ export interface Context {
   contextActivities?: ContextActivities;
   revision?: string;
   platform?: string;
-  language?: string;
+  language?: Exclude<string, 'und' | 'Und' | 'UND'>;
```
Comment (thomasturrell) [inline projects/ngx-xapi/model/src/lib/statement.ts:28]: Is there a locale type?

File: projects/ngx-xapi/model/src/lib/statement.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { Agent } from './actor';
```
Comment (thomasturrell) [inline projects/ngx-xapi/model/src/lib/statement.ts:1]: See Sonar smell about duplicate import

File: projects/ngx-xapi/model/src/lib/statement.ts (hunk 2 above)
Comment (Selindek) [reply projects/ngx-xapi/model/src/lib/statement.ts:91]: no

Comment (thomasturrell) [reply projects/ngx-xapi/model/src/lib/statement.ts:90]: What about this?

https://www.npmjs.com/package/@types/locale

**Pull request merged**